bindings/nodejs/lib/utils/milestone-helper.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { Block, MilestonePayload } from '../types';
+import { Utils } from './utils';
+
+/**
+ * Compute a milestoneId from a milestone payload.
+ * @param payload The milestone payload.
+ * @returns The milestone id hex prefixed string.
+ */
+export function milestoneIdFromMilestonePayload(
+    payload: MilestonePayload,
+): string {
+    return Utils.milestoneId(payload);
+}
```

Comment:
This feels really useless, I'd just let them call our function

Reply:
sure